hs_composite_resource/models.py

```diff
@@ -474,32 +492,11 @@ def check_file_rename_or_move():
             # check source - see if the folder containing the file represents an aggregation
             try:
                 can_move = check_file_rename_or_move()
-                if not can_move:
-                    return can_move
+                return can_move
             except ObjectDoesNotExist:
-                pass
+                return True
```

Comment:
Is it correct to return True if ObjectDoesNotExist exception is raised here?

Reply:
Yes. The function ```check_file_rename_move()``` checking for any aggregation affected by this rename/move. If no aggregation is found for the path being renamed/moved, it raises ObjectDoesNotExist which means file rename/move is allowed (return True).